src/service-object.ts

```diff
@@ -91,8 +91,6 @@ export interface Methods {
   [methodName: string]: {reqOpts?: r.CoreOptions}|boolean;
 }
 
-export interface CreateOptions {}
-
```

Comment:
An empty interfaces are a bad idea. Looking at it, my intuition tells me that a `CreateOptions` object must be empty. In fact, TypeScript does not do excess property checks on empty interfaces https://github.com/Microsoft/TypeScript/issues/14606 . If the intent here is to allow a _nearly_ arbitrary object as input to `ServiceObject.create`, better to type it explicitly as such as in this PR.